layers/auto-completion/packages.el

```diff
@@ -269,7 +269,7 @@
         "Call `yas-expand' and switch to `insert state'"
         (interactive)
         (call-interactively 'aya-expand)
-        (evil-insert-state))
+        (unless holy-mode (evil-insert-state)))
```

Comment:
@justbur I don't they are different. That's why this behaviour annoys. I was using holy mode and after invoking "M-m i S e", I was suddenly put into hybrid mode. I think the proper UX is not to change the holy mode to hybrid mode as a side effect invoking this command.